renovate.json

```diff
@@ -0,0 +1,73 @@
+{
+  "extends": [
+    "config:js-lib",
+    ":pinOnlyDevDependencies",
+    ":automergePatch",
+    ":automergeMinor",
+    ":semanticCommits"
+  ],
+  "timezone": "America/Los_Angeles",
+  "schedule": ["after 3am", "before 9am"],
+  "commitMessageTopic": "{{depName}}",
+  "commitMessage": "{{{commitMessagePrefix}}} {{{commitMessageTopic}}} {{{commitMessageSuffix}}}",
+  "commitMessageBody": "{{{commitMessageAction}}} {{{commitMessageExtra}}}",
+  "packageRules": [
+    {
+      "packageNames": ["typescript", "tslint", "typedoc", "dtslint"],
+      "groupName": "typescript packages",
+      "groupSlug": "typescript",
+      "minor": {
+        "automerge": false
+      }
+    },
+    {
+      "packagePatterns": ["^@types/"],
+      "groupName": "ambient types",
+      "groupSlug": "ambient-types",
+      "minor": {
+        "automerge": false
+      }
+    },
```

Comment:
Ambient type packages are published in batch. In the case of `@types/ember`, as many as ~24 packages may update at the same time -- you'll want them all in a single PR, rather than hoping that they'll each be upgraded successfully in isolation